pkg/extensions/extension-search.go

```diff
@@ -1,45 +1,23 @@
-//go:build extended
-// +build extended
+//go:build search || ui_base
```

Comment:
Do you think it makes sense to have 2 separate tags here? What's the difference between them?

Reply:
search - used for making zot binary with search extension
ui_base - used for making zli
This is why I chose in the first place to use 2 different tags, to not mix up their executable binaries / test binaries.